solutions/14.js

```diff
@@ -0,0 +1,14 @@
+// finds number in diagnalArray always cubed.
+// Phillip Kelley-Dotson
+
+const diagnalArray = (Array) => {
```

Comment:
@pkdotson please rename to solution, export as solution, and import as solution on the test file